core/frontend/src/test/render/GraphicBuilder.test.ts

```diff
@@ -248,7 +248,8 @@ describe("GraphicBuilder", () => {
         expect(builder.wantEdges).to.equal(generateEdges ?? true);
         addToGraphic(builder);
 
-        builder.finish();
+        const gf = builder.finish();

```

Comment:
Merge issue?

Reply:
I encountered an issue a few days ago where the lack of `.dispose()` was causing the mac cover tests to fail. I am uncertain why it started to fail or why it was only on mac, but this change solved it and matches the other tests in this file.

Reply:
Apple's WebGL implementation is flaky and sometimes extremely slow causing tests to time out.